README.md

```diff
@@ -290,6 +290,7 @@ Langchain::Loader.load('https://www.example.com/file.pdf')
 | html   | Processors::HTML | `gem "nokogiri", "~> 1.13"`  |
 | pdf    | Processors::PDF  | `gem "pdf-reader", "~> 1.4"` |
 | text   | Processors::Text |                              |
+| csv    | Processors::CSV  |                              |
```

Comment:
Should this say `Langchain::Processors::CSV` since you can't call `Processors::CSV` directly?

Reply:
Yeah, probably. Will update this table